tabula/util.py

```diff
@@ -148,7 +148,7 @@ class TabulaOption:
     stream: bool = False
     password: Optional[str] = None
     silent: Optional[bool] = None
-    columns: Optional[Sequence[float]] = None
+    columns: Optional[Union[Sequence[float], Sequence[Sequence[float]]]] = None
```

Comment:
Great! I didn't know multiple areas can be passed.

https://github.com/tabulapdf/tabula-java/blob/2cdf3b4fd3f7e921dca8cc6814cdd9316be40f0f/src/test/java/technology/tabula/TestCommandLineApp.java#L158-L160